tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -612,7 +612,7 @@ STATIC_ASSERT(__cpp_lib_constexpr_vector == 201907L);
 #endif
 #endif
 
-#if _HAS_CXX20 && defined(__cpp_impl_coroutine) // TRANSITION, Clang coroutine support
+#if _HAS_CXX20 && defined(__cpp_impl_coroutine) // TRANSITION, Clang 14 coroutine support

```

Comment:
The test does not cover the `/std:c++14 /await:strict` case, otherwise I'd expect this to fail because `__cpp_lib_coroutine` is defined.

Reply:
True - I think it suffices to remove the `_HAS_CXX20 &&` here. That said, I'm never sure how far we should go out of our way to verify non-standard modes. I'll go ahead and approve, and let STL make the call on adding down-level `/await:strict` coverage or not.

Reply:
Great catch! This test's `env.lst` adds extra configurations to exercise non-standard modes like `/Zc:noexceptTypes-` and `/D_HAS_STD_BYTE=0`. I can update it.

Reply:
Wow, `env.lst` had more dead code than I realized. Cleaning...